docs/management/admin/endpoint-protection-rules.asciidoc

```diff
@@ -0,0 +1,37 @@
+[[endpoint-protection-rules]]
+= Endpoint protection rules
+
+Endpoint protection rules are prebuilt rules designed to help you manage and respond to alerts generated by {elastic-endpoint}, the installed component that performs {elastic-defend}'s threat monitoring and prevention. These rules include the <<endpoint-security, {elastic-defend}>> rule as well as additional detection and prevention rules for different {elastic-defend} protection features.
+
+NOTE: To receive {elastic-endpoint} alerts, you must install the Endpoint agent on your hosts (see <<install-endpoint>>).
+
+[discrete]
+== {elastic-defend} rule
+
+The {elastic-defend} rule automatically creates an alert from all incoming {elastic-endpoint} alerts. When this rule is enabled, the following Endpoint events are displayed as detection alerts:
```

Comment:
The first sentence is a little confusing, probably because I don't know much about the Defend integration and how it interacts with Endpoint to generate alerts. Am I understanding correctly that the Elastic Defend rule creates alerts from Endpoint alerts, and not from Endpoint _events_? (Same question for [this line](https://github.com/elastic/security-docs/pull/6100/files#diff-2660cdd6a483bf299687cd255fea85908e3e47c4e2aad201cb73f4c193aaec29R11) in the Serverless docs.)